### DISCUSSION TITLE
maven repository

### DISCUSSION THREAD

**Discussion by nmelnic** [zalopay-oss/jmeter-grpc-request#69] [Ideas] [answered]
Hi, 

Any plan to upload the plugin/artifact to maven repository?

Currently I tried to see if it exists but got the following error:

[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.lazerycode.jmeter:jmeter-maven-plugin:3.4.0:configure (configuration) on project test_framework Could not find artifact vn.zalopay.benchmark:jmeter-grpc-request:jar:1.1.1 in centra
l (https://repo.maven.apache.org/maven2) -> [Help 1]

**Comment by anhldbk** [comment]
@huynhminhtan @thoainguyen We can publish this package to this Github registry ;)

**Comment by huynhminhtan** [comment, marked as answer]
Hi @nmelnic,

Yeh, of course. We will try to push to GitHub Packages (or some think like that) as soon as possible.
Thank anh @anhldbk for your suggestion.

**Reply by nmelnic** [reply]
Hi @huynhminhtan 

Any update on this request please? We would really appreciate if you can do this as soon as posible.

Just to give you some background on our issue, we are using this plugin for some of our JMeter test that are executed from Jenkins. Also we are using https://github.com/jmeter-maven-plugin/jmeter-maven-plugin for JMeter plugin/dependency management. All of our required dependencies are set into maven POM except jmeter-grpc-request, therefore we need it to be published into maven central repository as well: https://mvnrepository.com/repos/central in order to make our grpc test packs workable from ci/cd.

Is this request doable for you? and also if there is an ETA?

**Reply by nmelnic** [reply]
@huynhminhtan,  @thoainguyen agree with you, with GitHub Package Registry it would be  more easier and less time consuming in this case then using Maven.
